content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,101 @@
+---
+reviewers:
+title: RBAC Good Practices
+description: >
+  Principles and practices for good RBAC design for cluster operators.
+content_type: concept
+weight: 10
+---
+
+# Kubernetes RBAC Good Practices
+
+<!-- overview -->
+
+Kubernetes RBAC is a key security control to ensure that cluster users and workloads have only the access to resources required to execute their roles. It is important to ensure that, when designing permissions for cluster users, the cluster administrator understands the areas where privilge escalation could occur, to reduce the risk of excessive access leading to security incidents.
+
+The good practices laid out here should be read in conjunction with the general [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update).
+
+<!-- body -->
+
+## General Good Practice
```

Comment:
nit: page headings should be in sentence case

Reply:
Sentence cased the titles